### ISSUE TITLE
Structure the groups variable to be compatible with cellprofiler --print-groups

### ISSUE THREAD

**Issue by shntnu** [CellProfiler/Distributed-CellProfiler#52]
`cellprofiler --print-groups` outputs groups, which can be piped in to `jq 'map(.[0])' -' to produce output that looks like this
```
[
  {
    "Metadata_Plate": "Plate1",
    "Metadata_Well": "A01",
    "Metadata_Site": "1"
  },
  {
    "Metadata_Plate": "Plate1",
    "Metadata_Well": "A01",
    "Metadata_Site": "2"
  }
]
```

It would be great if the `groups` variable in the job config file used the format above

Currently, the format is: 
```
"groups": [
    {"Metadata": "Metadata_Plate=Plate1,Metadata_Well=A01,Metadata_Site=1"},
    {"Metadata": "Metadata_Plate=Plate1,Metadata_Well=A01,Metadata_Site=2"}
  ]
```

**Comment by bethac07** [comment]
Is there a reason why `cellprofiler --print-groups` would be the preferred way to create the groups list?  The advantage to the way we do it now (either working from the CSVs or just creating them iteratively with parallel) is that you don't actually ever have to load the images into CP- you have your CSV (which you can create in CP or not as your preference), then you just associate it with a pipeline, meaning that 

* it's easy to reuse the same pipeline over on a new data set with no additional step to load in images.  
* it's easy to run multiple pipelines on the same data set because you can just reuse the CSV and populate a new metadata list if necessary, again without having to actually do anything with the images.

It's not a bad idea to support multiple formats but I'm not sure how often this one would be used in practice.  Definitely open to counter arguments though 😄

**Comment by shntnu** [comment]
Beth and I chatted about this in person. My notes 

- I prefer going via CP because the groups will the same as defined in the pipeline
- This makes it easier to process one plate at a shot, which is the way to go (rather than several plates, or a fraction of a plate)
- There is an overhead but it is constant time and doesn't require interaction

For now, we've decided not to change the parser because it will need extensive testing. Instead, I might right a script that converts it into the DCP format

**Comment by bethac07** [comment]
Closed via #59
